### DISCUSSION TITLE
How can I sponsor this work?

### DISCUSSION THREAD

**Discussion by haznai** [Taitava/obsidian-shellcommands#427] [Q&A] [answered]
### Obsidian debug information

-

### What shell are you using?

Bash

### Please describe the problem

this is quite literally the quintessential plugin to me. the plain text manipulation + LLM prompts you can create with this plugin is amazing, awesome variability. making obsidian a citizen of the unix terminal is a genius idea.

anyway, i'd like to chip in some small amounts of money for this open-source project. (how) can i do that?

**Comment by Taitava** [comment, marked as answer]
Hi! Sorry for my absence.

Thank you for your kind feedback! ❤️ I appreciate your willingness to sponsor this project. However, I'm not currently taking donations. But if you or anyone are willing to donate to charity, I'd be happy for any [donations to Ukraine](https://publish.obsidian.md/shellcommands/Donate) that is suffering from a war. I'm not a Ukrainian myself, but I'd like to help them.
